src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -61,6 +61,7 @@
 
     public enum Feature {
         SWITCH_PATTERN_MATCHING,
+        UNIVERSAL_TVARS,
```

Comment:
As Dan noted yesterday, for other Valhalla features we do not generally use preview mechanisms (this code needs to be pushed in the valhalla repo). We can postpone the preview dance for later. In fact, to ease experiments in JDK code it would be better to just make universal type variable syntax available by default.

There are some other places where this UNIVERSAL_TVARS constant is used which should be cleaned up for now (e.g. Preview.java).

Reply:
then I guess we will need to update the text in the JEP as the it is explicitly saying that this one is a preview feature